changelogs/fragments/60076-get_certificate-add_option_proxy.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - Added proxy option to get_certificate. When using the proxy option, python 2.7 or higher is required.
```

```suggestion
  - get_certificate - added ``proxy_*`` options.
```
The details about Python 2.7 don't need to be in here IMO. Makes the changelog easier to read (especially since most people aren't using Py 2.6 anymore).

---

Thank you for pointing out.
Modified the changelog file.